app/views/full-page-examples/bank-holidays/index.njk

```diff
@@ -50,9 +50,10 @@ notes: >-
       {% set englandAndWalesHTML %}
 
         {% set englandAndWalesPanelTitleHtml %}
-          <div class='govuk-!-font-size-36 govuk-!-margin-bottom-4'>
+          <span class='govuk-!-font-size-36'>
             The next bank holiday in England and Wales is
-          </div>
+          </span>
+          <br><br>
```

Comment:
Is there a way to do this without the <br>? do we have a `govuk-!-display-block`? Could make an `app-` version maybe.

Reply:
Yes a good idea 👍